apiserver/client/status.go

```diff
@@ -172,8 +172,14 @@ func (c *Client) FullStatus(args params.StatusParams) (params.FullStatus, error)
 	var noStatus params.FullStatus
 	var context statusContext
 	var err error
+	if context.model, err = c.api.stateAccessor.Model(); err != nil {
+		return noStatus, errors.Annotate(err, "could not fetch model")
+	}
+	if context.status, err = context.model.LoadModelStatus(); err != nil {
+		return noStatus, errors.Annotate(err, "could not load model status values")
```

Comment:
Would we want to do this contextually on "len(args.Patterns) <= 0" ? I'm just wondering if "juju status foo" is going to have to load the status of everything first.